lib/reboot_cmds.py

```diff
@@ -139,21 +138,14 @@ def main(cfg):
     :raises: LockHeldError when lock still held by auto-attach after retries.
              UserFacingError for all other errors
     """
-    while True:
-        try:
+    try:
+        with lock.SpinLock(
```

Comment:
What do you think if we used named parameters for the locks ? Just to make reading the code easier and just to not rely on the parameter order to be correct for every call

Reply:
ohhh very good idea I'll do that